config/dictionaries/resource.json

```diff
@@ -228,6 +228,7 @@
       "note",
       "local_id",
       "status",
+      "overall_status",
```

Comment:
Is there a way to specify that this field only appears if explicitly `$select`ed, perhaps in the examples?

Reply:
That could be a good one, I have been confused by the CLI's output of device status where it show is_online === true for even devices that are offline. @lmbarros Let's push to get this merged and add an example if you can.

Reply:
@cywang117 , @vipulgupta2048 I added a commit with the example. We could maybe have a more inspiring `description` field, perhaps giving some info about it is and how `overall_status` differs from `status`. I think I don't grok this to the point of being able to come with a correct and concise description, though. Feel free to suggest!

Reply:
How about we point back to the docs to explain overall_status? 
I don't know what `status` is about @lmbarros

Reply:
I don't think `overall_status` is documented, @vipulgupta2048 :-/ My understanding is that it is on the same spirit as the status on the dashboard (Thodoris described it as "the device state from the observer's/user's point of view"), but it is not the same logic as the one used on the dashboard, with different (and undocumented) possible values, IIUC.

Reply:
[A bit more context](https://balena.zulipchat.com/#narrow/stream/345882-_help/topic/Computed.20connectivity.20state.20SDK.20method) on a thread from less than an hour ago, by coincidence.